src/Http/Ownership/RequestOwnershipRequestHandler.php

```diff
@@ -73,6 +79,18 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             );
         }
 
+        // Make sure the organizer does exists
+        if ($requestOwnership->getItemType()->sameAs(ItemType::organizer())) {
+            try {
+                $this->organizerRepository->fetch($requestOwnership->getItemId()->toString());
+            } catch (DocumentDoesNotExist $e) {
+                throw ApiProblem::organizerNotFound($requestOwnership->getItemId()->toString());
+            }
+        }
+
+        // TODO: Check if owner exists
```

Comment:
The nitiest of the nitpicks: if you start todo's with an @todo you can fetch a list from phpstorm at the left bar at the bottom. :-D